PusherClient/Pusher.cs

```diff
@@ -130,7 +130,7 @@ public Channel Subscribe(string channelName)
             if (_connection.State != ConnectionState.Connected)
                 throw new PusherException("You must wait for Pusher to connect before you can subscribe to a channel", ErrorCodes.NotConnected);

```

Comment:
There probably are reasons that making this change would break things. Maybe around a Channel actually being in a `Subscribing` state - something we don't have right now.

So, let's ignore my comment and I'll manually test as-is and merge etc.

Thanks :+1: